libbpf-rs/src/map.rs

```diff
@@ -133,6 +133,17 @@ impl Map {
         }
     }
 
+    /// Returns if the map is of one of the per-cpu types.
+    pub fn is_percpu(&self) -> bool {
+        match self.map_type() {
+            MapType::PercpuArray
+            | MapType::PercpuHash
+            | MapType::LruPercpuHash
+            | MapType::PercpuCgroupStorage => true,
+            _ => false,
+        }
+    }
```

Comment:
I wonder if:
* this should be a method on `MapType` 
* we should add a `map_type()` method on `OpenMap` as well

so the OpenMap and Map are symmetric.

Thoughts?

Reply:
Making this a method on MapType sounds like a good idea. I'll move it over there. 

I am not so sure about implementing `map_type()` on `OpenMap`. It seems like we don't have a lot of information available at that stage and it feels a bit out of scope for this PR. But if `is_percpu()` is on `MapType`, it is immediately usable on `OpenMap` if someone implements `map_type()` there.